src/shared/dropdown-list-v2/dropdown-list.styles.tsx

```diff
@@ -188,3 +196,12 @@ export const Spinner = styled(ComponentLoadingSpinner)`
     margin-right: ${Spacing["spacing-8"]};
     color: ${Colour["icon"]};
 `;
+
+export const NoResultDescContainer = styled.div<ContainerStyleProps>`
```

Comment:
suggest to include `applyHtmlContentStyle` so that the description can automatically have HTML styling (see [Markup](https://designsystem.life.gov.sg/react/index.html?path=/docs/core-markup--docs))